tools/generators/dependency-mismatch/index.spec.ts

```diff
@@ -159,36 +170,74 @@ describe('dependency-mismatch generator', () => {
         [`${workspaceNpmScope}/tslib`]: '^2.1.1',
       },
       devDependencies: {},
+      peerDependencies: {},
     });
 
+    await generator(appTree);
+
     const packageJson: PackageJson = await readTargetPackageJson();
     expect(packageJson.dependencies).toMatchInlineSnapshot(`
       Object {
         "proj/tslib": "^2.1.1",
       }
     `);
   });
+
+  it('should ignore northstar packages', async () => {
+    const { readPackageJson: readTargetPackageJson } = setupDummyPackage(
+      appTree,
+      {
+        name: 'react-northstar',
+        version: '0.66.0',
+        dependencies: {
+          [`${workspaceNpmScope}/dom-utilities`]: '^1.1.1',
+        },
+        devDependencies: {},
+        peerDependencies: {},
+      },
+      ['northstar'],
+    );
+
+    setupDummyPackage(appTree, {
+      name: 'dom-utilities',
+      version: '2.1.2',
+      dependencies: {},
+      devDependencies: {},
+      peerDependencies: {},
+    });
+
+    await generator(appTree);
+
+    const packageJson: PackageJson = await readTargetPackageJson();
+    expect(packageJson.dependencies).toMatchInlineSnapshot(`
+      Object {
+        "proj/dom-utilities": "^1.1.1",
+      }
+    `);
+  });
 });
 
 function setupDummyPackage(
   tree: Tree,
-  options: {
+  packageJson: {
     name: string;
     version: string;
     devDependencies: Record<string, string>;
     dependencies: Record<string, string>;
+    peerDependencies: Record<string, string>;
   },
+  tags: string[] = [],
```

Comment:
**chore:** can we have this only as options (named arguments), positional arguments don't scale and provide bad DX on call site